src/app.js

```diff
@@ -0,0 +1,79 @@
+import axios from "axios";
+
+
+// Referenties naar elementen in HTML document toewijzen aan variabelen
+const countryDetailsField = document.getElementById("country-details-field");
+const countryNameInput = document.getElementById("country-name-input-field");
+const countryNameInputForm = document.getElementById("country-name-input-form");
+const errorMessage = document.getElementById("error-message");
+
+
+// URI aan (final) variabele toewijzen
+// Lege variabele declareren om later de endpoint aan toe te kunnen wijzen
+const URI = "https://restcountries.com/";
```

Comment:
Variabelen in camelCase schrijven
```suggestion
const uri = "https://restcountries.com/";
```